walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,71 @@
 
 package com.google.sps;
 
+import static java.util.Collections.EMPTY_LIST;
+import static java.util.Collections.disjoint;
+import static java.util.Collections.sort;
+import static java.util.Arrays.asList;
+
+import java.util.Comparator;
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.List;
+
 
+// FindMeetingQuery finds open meeting timeslots throughout the day.
```

Comment:
for commenting at class or method level use /** */ notation.

Reply:
Done.